### PR TITLE
6.14.10

### PR DESCRIPTION
**Pull request by darcyclarke**
### DEPENDENCIES

* [`906d647e1`](https://github.com/npm/cli/commit/906d647e1cacd74243abcacba3bade80437f30f5) `opener@1.5.2`
  * fixes: [`#36445`](https://github.com/nodejs/node/issues/36445) addressing [`GHSL-2020-145`](https://securitylab.github.com/advisories/GHSL-2020-145-domenic-opener)

### CODE REVIEW

**Review by ruyadorno — APPROVED** [review]

File: AUTHORS [hunk 1]
```diff
@@ -708,3 +708,4 @@ Sandra Tatarevićová <sandra@tatarevicova.cz>
 Antoine du Hamel <duhamelantoine1995@gmail.com>
 Assaf Sapir <assapir@github.com>
 Lukas Spieß <lumaxis@github.com>
+Darcy Clarke <darcyclarke@GH.local>
```
Comment (ruyadorno) [inline AUTHORS]: oops, you might want to fix this actually

**Comment by npm-deploy-user** [comment]
<table>
    <thead>
      <tr>
        <th></th>
        <th colspan="3">angular-quickstart</th>
<th colspan="3">app-large</th>
<th colspan="3">app-medium</th>
<th colspan="3">ember-quickstart</th>
<th colspan="3">react-app</th>
      </tr>
      <tr>
        <th></th>
        <th>prev</th>
<th>current</th>
<th>status</th>
<th>prev</th>
<th>current</th>
<th>status</th>
<th>prev</th>
<th>current</th>
<th>status</th>
<th>prev</th>
<th>current</th>
<th>status</th>
<th>prev</th>
<th>current</th>
<th>status</th>
      </tr>
    </thead>
    <tbody>
      <tr>
<td>initial install</td>
<td>41s</td>
<td>33.9s</td><td>✅</td>
<td>39.5s</td>
<td>38.9s</td><td>✅</td>
<td>34.5s</td>
<td>31.3s</td><td>✅</td>
<td>28.1s</td>
<td>25.6s</td><td>✅</td>
<td>33.5s</td>
<td>30.6s</td><td>✅</td>
</tr>
<tr>
<td>repeat install</td>
<td>9.6s</td>
<td>8.5s</td><td>✅</td>
<td>8.6s</td>
<td>7.8s</td><td>✅</td>
<td>8.3s</td>
<td>7.1s</td><td>✅</td>
<td>7.7s</td>
<td>6.3s</td><td>✅</td>
<td>9.1s</td>
<td>8s</td><td>✅</td>
</tr>
<tr>
<td>with warm cache</td>
<td>32.3s</td>
<td>28.3s</td><td>✅</td>
<td>33.8s</td>
<td>32.4s</td><td>✅</td>
<td>31.3s</td>
<td>28.2s</td><td>✅</td>
<td>23.4s</td>
<td>19.9s</td><td>✅</td>
<td>29.1s</td>
<td>27.1s</td><td>✅</td>
</tr>
<tr>
<td>with node_modules</td>
<td>9.2s</td>
<td>7.8s</td><td>✅</td>
<td>8.6s</td>
<td>7.3s</td><td>✅</td>
<td>9s</td>
<td>7.3s</td><td>✅</td>
<td>7.8s</td>
<td>5.8s</td><td>✅</td>
<td>9.4s</td>
<td>7.2s</td><td>✅</td>
</tr>
<tr>
<td>with lockfile</td>
<td>32.6s</td>
<td>26.9s</td><td>✅</td>
<td>31s</td>
<td>30.4s</td><td>✅</td>
<td>29s</td>
<td>26.2s</td><td>✅</td>
<td>21.7s</td>
<td>19.8s</td><td>✅</td>
<td>27.4s</td>
<td>24.7s</td><td>✅</td>
</tr>
<tr>
<td>with warm cache and node_modules</td>
<td>9.3s</td>
<td>8.1s</td><td>✅</td>
<td>7.9s</td>
<td>7s</td><td>✅</td>
<td>8.4s</td>
<td>6.6s</td><td>✅</td>
<td>7.6s</td>
<td>6.2s</td><td>✅</td>
<td>9.2s</td>
<td>7.5s</td><td>✅</td>
</tr>
<tr>
<td>with warm cache and lockfile</td>
<td>24.7s</td>
<td>22.4s</td><td>✅</td>
<td>26.3s</td>
<td>27.8s</td><td>✅🐌</td>
<td>24.3s</td>
<td>19.2s</td><td>✅</td>
<td>17.8s</td>
<td>15s</td><td>✅</td>
<td>21.3s</td>
<td>20.7s</td><td>✅</td>
</tr>
<tr>
<td>with node_modules and lockfile</td>
<td>10.1s</td>
<td>8.8s</td><td>✅</td>
<td>9.8s</td>
<td>8.2s</td><td>✅</td>
<td>8.6s</td>
<td>7.2s</td><td>✅</td>
<td>7.8s</td>
<td>6.7s</td><td>✅</td>
<td>9.7s</td>
<td>8.5s</td><td>✅</td>
</tr>
    </tbody>
  </table>

**Comment by ljharb** [comment]
Reminder to get this into node 10/12/14 ASAP, there's a security release coming soon

**Comment by darcyclarke** [comment]
@ljharb that's the plan is to open a PR today

**Review by nlf — APPROVED** [review]
:+1:

**Review by sa-orlov — COMMENTED** [review]
Ik

**Review by sa-orlov — COMMENTED** [review]
Ok

**Pull request merged**